app/lib/index.js

```diff
@@ -338,7 +338,10 @@ app.createEditorWindow = function() {
   const windowOptions = {
     resizable: true,
     show: false,
-    title: 'Camunda Modeler'
+    title: 'Camunda Modeler',
+    webPreferences: {
+      preload: path.join(__dirname, 'preload.js')
```

Comment:
For the current version of electron, we should explicitly disable node integration. At the moment, the PR still allows to run `require('electron').remote.require('fs')`.
```suggestion
      preload: path.join(__dirname, 'preload.js'),
      nodeIntegration: false
```

Reply:
Good catch. Absolutely required. I did simply forget about it :+1: